hvplot/converter.py

```diff
@@ -229,7 +232,8 @@ class HoloViewsConverter(object):
                      'min_height', 'min_width', 'frame_height', 'frame_width',
                      'aspect', 'data_aspect']
 
```

Comment:
you mentioned plot_option in you note, but do you mean it shouldn't be in _style_options, but instead in _axis_options?